content/en/docs/tasks/administer-cluster/reserve-compute-resources.md

```diff
@@ -96,7 +96,7 @@ system daemon should ideally run within its own child control group. Refer to
 for more details on recommended control group hierarchy.
 
 Note that Kubelet **does not** create `--kube-reserved-cgroup` if it doesn't
-exist. Kubelet will fail if an invalid cgroup is specified. With `systemd`
+exist. `kubelet` will fail if an invalid cgroup is specified. With `systemd`
```

Comment:
Here's how the style guide recommends how to do it (I didn't check, this is my recollection):
```suggestion
Note that the kubelet **does not** create `--kube-reserved-cgroup` if it doesn't
exist. The kubelet will fail to start if an invalid cgroup is specified. With `systemd`
```